library-no-op/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -13,19 +13,19 @@ object Chucker {
 
     val isOp = false
 
-    @JvmStatic fun getLaunchIntent(context: Context, screen: Int): Intent {
```

Comment:
Ideally we should keep those `@JvmStatic`, at least for all the methods inside `com.chuckerteam.chucker.api`.

Removing this annotation will generate a breaking change + will make Chucker less Java-friendly.

Reply:
Yeah, somehow thought that it is only fo internal usages. Reverting.